daemon/config_unix.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	EnableCors           bool
 	EnableSelinuxSupport bool
 	SocketGroup          string
+	Sysctls              map[string]string
```

Comment:
No removed.  Patch built off of Ulimits patch and don't think we need options for the daemon.  Left this by mistake.